src/index.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+const App = () => (
+  <div>
+      <h1>Hello world!!</h1>
+  </div>
+ )
+ ReactDOM.render(<App/>, document.getElementById('root'));
```

Comment:
You are calling `<App/>` component yet you commented it out, can you throw more light on this please ?

Reply:
I have uncommented the code contained in the App component.